src/main/java/com/arkivanov/gradle/SetupJsApp.kt

```diff
@@ -2,11 +2,11 @@ package com.arkivanov.gradle
 
 import org.gradle.api.Project
 import org.gradle.kotlin.dsl.configure
-import org.jetbrains.kotlin.gradle.dsl.KotlinJsProjectExtension
+import org.jetbrains.kotlin.gradle.dsl.KotlinMultiplatformExtension
 
 fun Project.setupJsApp() {
-    extensions.configure<KotlinJsProjectExtension> {
-        js(IR) {
+    extensions.configure<KotlinMultiplatformExtension> {
```

Comment:
Do we really need to do this change now?

Reply:
I'm not sure. Usually I'm trying to remove usages of deprecated APIs whenever possible. I can revert this change, for sure.

Reply:
Reverted this one